src/data.js

```diff
@@ -672,6 +672,26 @@ let WorkerType = Entity.configure({
     // the time at which this worker-type should no longer be displayed
     expires:          Entity.types.Date,
   },
+}).configure({
+  version:            2,
+  properties: {
+    provisionerId:    Entity.types.String,
+    workerType:       Entity.types.String,
+    // the time at which this worker-type should no longer be displayed
+    expires:          Entity.types.Date,
+    description:      Entity.types.String,
+    stability:        Entity.types.String,
+    documentation:    Entity.types.String,
+    payloadSchema:    Entity.types.JSON,
```

Comment:
Docs and schema should not be in azure-entities, S3 or blob storage is probably best